packages/connect/CHANGELOG.md

```diff
@@ -10,6 +10,57 @@
 
 Use the persistent link [connect.trezor.io/9](https://connect.trezor.io/9/) to access the latest stable version of Connect Explorer.
 
+# 9.3.1-beta.1
+
+Remove support for legacy passprahse support (7bb57cf)
+authenticateDevice verify CA cert extensions (4ba4445)
+
+    - chore(connect): update of t3b1 minimal requirements (f7f1705)
```

Comment:
"Add support for T3B1" ?

Reply:
[5a4ec63](https://github.com/trezor/trezor-suite/pull/13517/commits/5a4ec63bcce90878cdaf45aebac440a34a6752c6)